templates/inspec/examples/google_compute_ssl_policies.erb

```diff
@@ -5,4 +5,11 @@ describe resource do
   it { should exist }
   its('names') { should include <%= doc_generation ? "'#{ssl_policy['name']}'" : "ssl_policy['name']" -%> }
   its('profiles') { should include <%= doc_generation ? "'#{ssl_policy['profile']}'" : "ssl_policy['profile']" -%> }
+  its('count') { should eq 1 }
+end
+
+resource.names.each do |policy_name|
```

Comment:
Fine for now, but this looks really separate. Probably should be a separate PR in the future.

Reply:
True.